src/components/ShoppingCart/ShoppingCartContainer.tsx

```diff
@@ -0,0 +1,80 @@
+import { Table, Thead, Tbody, Tr, Th, TableContainer, Text, Button, Flex } from "@chakra-ui/react";
+import CartListItem from "./CartListItem";
+import { ICartItem } from "@/interfaces/cartItem";
+
+interface userCartList {
+  userShoppingCart: ICartItem[];
+}
+
+const ShoppingCartContainer = ({ userShoppingCart }: userCartList) => {
```

Comment:
The name of this prop can be a general name without the prefix "use" word.
Prefix "use" normally for hooks and custom hooks but not for props.

Reply:
> The name of this prop can be a general name without the prefix "use" word. Prefix "use" normally for hooks and custom hooks but not for props.

The prop "userShoppingCart" was renamed as "shoppingCart"